scripts/test_uitls.ts

```diff
@@ -0,0 +1,12 @@
+export const getSystemTestAccounts = (): string[] =>
+	JSON.parse(process.env.WEB3_SYSTEM_TEST_ACCOUNTS ?? '[]') as string[];
+
+export const getSystemTestProvider = (): string => process.env.WEB3_SYSTEM_TEST_PROVIDER ?? '';
+
+export const getSystemTestPassword = (): string => process.env.WEB3_SYSTEM_TEST_PASSWORD ?? '';
```

Comment:
is this being used anywhere in the codebase? should the default `WEB3_SYSTEM_TEST_PASSWORD` be [this](https://github.com/ChainSafe/web3.js/pull/4949/files#diff-5a4957bc7903de70fcfb50dc6aa8851f9d16d84566568736af52e9e2df8c8628R3)?

Reply:
It's used in the shell scripts to start ganache and geth.

Reply:
This specific function though doesn't seem to be called anywhere in this PR though